FEWD/Week03/lesson06/homework/startup_matchmaker/style/style.css

```diff
@@ -0,0 +1,243 @@
+* {
+  box-sizing: border-box;
+}
+
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after {
+  clear: both;
+}
+
+/*HEADER, BODY, FOOTER OF PAGES*/
+
+body {
+  width: 100%;
+  font-family: "Source Sans Pro", helvetica, sans-serif;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+}
+
+header {
+  color: white;
+  background-color: #000;
+  height: 75px;
+  border-bottom: 5px solid #ffff1a;
+}
+
+h1 {
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 25px;
+  float: left;
+  font-family: Merriweather;
+  padding-top: 5px;
+}
+
+nav {
+  float: right;
+  height: 75px;
+  padding: 20px 0;
```

Comment:
Ok so here, as opposed to your last HW, you __don't__ want the padding on the parent, the `<nav>` and that's because of the `:hover` styling on the individual `<a>`'s. You also don't want to limit the height.

Instead, you want to let the height of the nav be dictated by that of its children, the `<li>`'s, whose height in turn should be dictated by the `padding` on the `<a>`

In summary you want something along the lines of:

```
nav {
  float: right;
}

nav a { /* you're using "header li a" to target these links in your code on line 64, I'm just choosing some simpler CSS to target them */
  text-decoration: none;
  color: white;
  padding: 15px 10px;
  display: block; /* the key to letting you add padding to <a>'s */
}

nav a:hover {
  background-color: #b3b3b3;
}
```